topics/admin/tutorials/heterogeneous-compute/slides.html

```diff
@@ -2,13 +2,13 @@
 layout: base_slides
 logo: "GTN"
 
-title: "Using Heterogeneous Compute Resources"
+title: "Running Jobs on Remote Resources with Pulsar"
```

Comment:
While they can be remote systems they don't have to be.

I'd prefer to leave the original title / first section heading, but if you can think of a good way to say "heterogenous and/or remote", we can change to that?

Reply:
I think Pulsar should be on the title. What about "Using Pulsar to handle heterogeneous remote resources"?

Reply:
or better "Using Pulsar to handle heterogeneous compute resources"

Reply:
> Using Pulsar to handle heterogeneous compute resources

sounds great!